ws/ws.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/codequest-eu/octopussy"
+)
+
+func startServer(amqpUrl string) *octopussy.Server {
+	server := &octopussy.Server{
+		// TODO(janek): move to env variable?
+		Exchange: "ws_notification",
+		OnError: func(e error) {
+			log.Fatal(e.Error())
```

Comment:
`log.Fatal` will panic which will effectively shut down the server. Is it what we want?